### PR TITLE
bug/945_pagination_with_cprs

### PR DESCRIPTION
**Pull request by kzangeli**
Including 'Count: XXX' in ErrorCode::details also when CPrs are involved

### CODE REVIEW

File: src/lib/orionTypes/QueryContextResponseVector.cpp [hunk 1]
```diff
@@ -109,7 +109,7 @@ void QueryContextResponseVector::present(void)
 *
```
Comment (fgalan) [inline src/lib/orionTypes/QueryContextResponseVector.cpp]: CHANGES_NEXT_RELEASE entry missing.

Comment (kzangeli) [inline src/lib/orionTypes/QueryContextResponseVector.cpp]: Fixed in ad57d17

**Comment by fgalan** [comment]
LGTM. Waiting for LGTM from @anabelengp in order to merge.

**Comment by anabelengp** [comment]
LGTM

**Pull request merged**